README.md

```diff
@@ -21,9 +21,13 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-# 📖 **BLOG APP** <a name="about-project"></a>
+# 📖 **BUDGET BUDDY** <a name="about-project"></a>
 
-> A **BUDGET APP** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+> A **BUDGET BUDDY** is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+> To know more about the app, watch this [Video Presentation](https://drive.google.com/file/d/15xbmuU3o9ElexqBmzX12OQOQS2WAlvBB/view?usp=sharing) !
```

Comment:
Nice job walking us through your project with your video presentation😉. It is a requirement that you talk about a piece of code you have implemented and you are proud of😉. Kindly do well to talk about that as this makes your presentation more interactive and shows your amazing skills😉.

<img width="1055" alt="image" src="https://github.com/anita00001/budget-app/assets/64643995/ce214875-0e54-43d1-90de-c0195db298c9">